4-projects/business_unit_1/nonproduction/outputs.tf

```diff
@@ -108,3 +108,13 @@ output "iap_firewall_tags" {
   description = "The security tags created for IAP (SSH and RDP) firewall rules and to be used on the VM created on step 5-app-infra on the peering network project."
   value       = module.env.iap_firewall_tags
 }
+
+output "default_region" {
+  description = "The default region for the project."
+  value       = local.default_region
+}
+
+output "default_region" {
+  description = "The default region for the project."
+  value       = local.default_region
+}
```

Comment:
duplicated

Reply:
it also needs `make docker_generate_docs` to update the documentation on the README files:
```
Checking for documentation generation
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=autogen' '--exclude=*.tfvars' '--exclude=*metadata.yaml' /workspace/4-projects/business_unit_1/development/README.md /tmp/tmp.DTBdp0iX7D/generate_docs/workspace/4-projects/business_unit_1/development/README.md
22a23
> | default\_region | The default region for the project. |
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=autogen' '--exclude=*.tfvars' '--exclude=*metadata.yaml' /workspace/4-projects/business_unit_1/nonproduction/README.md /tmp/tmp.DTBdp0iX7D/generate_docs/workspace/4-projects/business_unit_1/nonproduction/README.md
22a23
> | default\_region | The default region for the project. |
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=autogen' '--exclude=*.tfvars' '--exclude=*metadata.yaml' /workspace/4-projects/business_unit_1/production/README.md /tmp/tmp.DTBdp0iX7D/generate_docs/workspace/4-projects/business_unit_1/production/README.md
22a23
> | default\_region | The default region for the project. |
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=autogen' '--exclude=*.tfvars' '--exclude=*metadata.yaml' /workspace/4-projects/business_unit_2/development/README.md /tmp/tmp.DTBdp0iX7D/generate_docs/workspace/4-projects/business_unit_2/development/README.md
22a23
> | default\_region | The default region for the project. |
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=autogen' '--exclude=*.tfvars' '--exclude=*metadata.yaml' /workspace/4-projects/business_unit_2/nonproduction/README.md /tmp/tmp.DTBdp0iX7D/generate_docs/workspace/4-projects/business_unit_2/nonproduction/README.md
22a23
> | default\_region | The default region for the project. |
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=autogen' '--exclude=*.tfvars' '--exclude=*metadata.yaml' /workspace/4-projects/business_unit_2/production/README.md /tmp/tmp.DTBdp0iX7D/generate_docs/workspace/4-projects/business_unit_2/production/README.md
22a23
> | default\_region | The default region for the project. |
Error: Documentation generation has not been run, please run the
```

Reply:
fixed in 40cbeb37596a42f796ea9613009cd3e297d128b1 and 358f1e0032e2ff0efbc0ef00f0284a43ce036ab9